### ISSUE TITLE
Crash when app start: java.lang.RuntimeException: Unable to get provider ...

### ISSUE THREAD

**Issue by MaximeLimone** [apache/cordova-android#574]
Hello, 

I have a fatal exception when i build the application with cordova-android 7.1.3. 

 ```
 java.lang.RuntimeException: Unable to get provider de.appplant.cordova.plugin.notification.util.AssetProvider: java.lang.ClassNotFoundException: Didn't find class "de.appplant.cordova.plugin.notification.util.AssetProvider" on path: DexPathList[[zip file "/data/app/com.esker.mobile.app-BLAUWYcRqHp1vyyns745jw==/base.apk"],nativeLibraryDirectories=[/data/app/com.esker.mobile.app-BLAUWYcRqHp1vyyns745jw==/lib/arm64, /system/lib64, /vendor/lib64]]
        at android.app.ActivityThread.installProvider(ActivityThread.java:6493) 
```

You can see the complete exception on the image.

The plugin who cause this issue seems to be **Cordova-plugin-notification**.

![capture](https://user-images.githubusercontent.com/6944958/48911568-53d64080-ee73-11e8-9852-7d3559663f22.JPG)

**Comment by janpio** [comment]
Duplicate of https://github.com/apache/cordova-android/issues/570

We already fixed it and there will be a 7.1.4 release as fast as possible. 
Until then, stay at 7.1.2. Thanks.

**Comment by brodybits** [comment]
(Fixed in PR #571)

**Comment by MaximeLimone** [comment]
Thank you that what we did ;)

**Comment by brodybits** [comment]
Patch release is still waiting for one more vote [[1]](https://lists.apache.org/thread.html/a1cd03b0a8c85539cbd370c0f77957c262b364085172a22ea9eb5249@%3Cdev.cordova.apache.org%3E)

[1] <https://lists.apache.org/thread.html/a1cd03b0a8c85539cbd370c0f77957c262b364085172a22ea9eb5249@%3Cdev.cordova.apache.org%3E>

**Comment by brodybits** [comment]
Should be resolved in 7.1.4.
